src/openms/CMakeLists.txt

```diff
@@ -130,6 +131,7 @@ openms_add_library(TARGET_NAME  OpenMS
                                      ${SQLITE_INCLUDE_DIR}
                                      ${LIBSVM_INCLUDE_DIRS}
                                      ${GLPK_INCLUDE_DIRS}
+				     ${KISSFFT_INCLUDE_DIR}
```

Comment:
in github it looks like a space character is missing before  ${KISSFFT_INCLUDE_DIR} or different tab settings have been used